lua/lazyvim/plugins/extras/coding/codecompanion.lua

```diff
@@ -0,0 +1,47 @@
+return {
+  {
+    "olimorris/codecompanion.nvim",
+    cmd = { "CodeCompanion", "CodeCompanionActions", "CodeCompanionToggle", "CodeCompanionAdd", "CodeCompanionChat" },
+
+    opts = function()
+      local options = require("codecompanion.config")
+      local user = vim.env.USER or "User"
+
+      options.strategies.chat.roles = {
+        llm = "  CodeCompanion",
+        user = "  " .. user:sub(1, 1):upper() .. user:sub(2),
```

Comment:
I don't think it should be forced into title case. Each person has chosen their user's name and in my case it looks weird to read it like that.
```suggestion
        user = "  " .. user,
```